Update outdated dialogue (#9417)
endless-sky/endless-sky: data/human/free worlds 4 epilogue.txt

					goto excitement
				`	"It's kind of hard to adjust to civilian life."`
					goto adjust

> 				`	"Is this it? Is the game over now that humanity's at peace again?"`

Reviewer: I edited my approval to also approve of lumbar527's suggestion, and offer a few other options. The following one is, I think, the best, but I don't know what EjoThims and Saugia have in mind:
```suggestion
				`	"I want to explore the rest of the galaxy, but apparently that part of the game hasn't been completed yet."`
```

Reviewer 2: I think reverting the change to "some parts" back to "that part" should be switched, it puts us back to possibly implying that "the rest of the galaxy" is a single whole unit that isn't completed at all, instead of specifically addressing "the rest of the galaxy" as distinct parts that each may or may not be completed.

But I don't have an issue with using "completed" instead of "implemented" here. 👍

Reviewer: Changing that to "some parts" is fine with me.

Reviewer 3: I'm a fan of this approach.

Reviewer 4: Which one? The original one in the comment, or the following one in reply to it?

Reviewer: I'm assuming the one in reply to it, because if it was the original one Amazinite probably would have responded to the whole PR and not this thread.

Reviewer 3: Ziproot's suggestion. It maintains the idea of the original choice while being more accurate to how the game is today. Sure, there's stuff implemented in the rest of the galaxy now--when this choice was added, all that existed was human space, Hai space, and the Korath exile systems--but it's not fully implemented in the sense that the regions that do exist are unfinished.

Reviewer 5: This suggestion (Ziproot's) is great!

> 				`	"Is this it? Is the game over now that humanity's at peace again?"`

Reviewer: ```suggestion
				`	"I want to explore the rest of the galaxy, but apparently that part of the game hasn't been fully implemented yet."`
```
See my reasoning and that of EjoThims above for why I chose this format.

Reviewer 6: I'd be happy with this.

> 				`	"Is this it? Is the game over now that I completed the free worlds campaign?"`

Reviewer 2: ```suggestion
				`	"I want to explore the rest of the galaxy, but apparently not all parts of the game have been implemented yet."``
```

Also, Free Worlds should be capitalized if this isn't reverted. ;)

Author: A new player may interpret "the rest of the galaxy" to mean "everything beyond human space" and "not all parts of the game have been implemented yet" to mean "everything beyond human space is not implemented yet"

Reviewer 7: Something like;
'I want to explore beyond human space, but apparently it hasn't all been implemented in the game yet,' perhaps?

> 				`	"Is this it? Is the game over now that I completed the free worlds campaign?"`

Reviewer 5: ```suggestion
				`	"I want to explore the rest of the galaxy, but I don't know what's been implemented."`
```
I'm just going to try suggesting something.

Author: @EjoThims, this suggestion does convey "I am excited for even more ES" without the risk of confusing the player. Is this a sufficiently obvious 4th wall break? If not, how about "I want to explore the rest of the galaxy, but I don't know which parts of the game are already implemented"?

Author: @Zireael07 How about this version?

Reviewer 2: This is significantly better than what you rolled out, especially in regards to at all actually matching up to Katya's response, but it's still drastically changing the meaning of the question itself, this time from an open ended but valid assumption to a presentation of ignorance from the player. It also loses the direct implication of continuing implementation from dropping that "yet." These together are the reason I didn't directly support it already as I did other continuing suggestions that I did feel were better.

A confused player is now directly being told that there's already more beyond human space in Katya's answer, though, and I still honestly don't think either this one or the one you committed to the PR actually addresses that issue any better, anyway.

Reviewer 8: I think it's slightly better than the previous one, and I agree mostly with @EjoThims

Author: > A confused player is now directly being told that there's already more beyond human space in Katya's answer

Not if the player selects option 1 "I miss the excitement..." or option 2 "It's kind of hard..."

Reviewer 2: > Not if the player selects option 1 "I miss the excitement..." or option 2 "It's kind of hard..."

Because neither of those are addressing the state of the game with a fourth wall break.

They shouldn't be given fourth wall breaking information about the game itself when they've chosen to avoid that.

> 				`	"Is this it? Is the game over now that I completed the free worlds campaign?"`

Reviewer 6: I am not a fan of the fourth wall breaks referring to "missions" and "campaigns" ... while technically true, it feels sort of cheesy and trivializes the whole story aspect, and just feels _wrong_. Also, how do you even define the end of FW? Are the epilogue campaigns part of it or not? I don't think a campaign should ever have a hard-defined end point.
```suggestion
				`	"Is this it? Is the game over now that humanity's at peace again?"`
```

> 				`	"I want to expand the game. Maybe add more missions or ships or alien species."`

Reviewer 4: ```suggestion
				`	"I want to explore the rest of the galaxy, but apparently not all of it has been implemented yet."`
```

> 				`	"I want to explore the rest of the galaxy, but apparently not all of it has been implemented yet."`

Reviewer 2: ```suggestion
				`	"I want to explore the rest of the galaxy, but apparently not all of the game has been implemented yet."`
```

Whole thing is still a 4th wall break, so we may as well leave the direct reference here.

Reviewer 4: I was considering this, but it seemed odd since the original was saying "that part of the game", when here, it's "the game", which are two different meanings.

> 				`	"I want to explore the rest of the galaxy, but apparently parts of it are not yet implemented in the current version of the game."`

Reviewer 5: ```suggestion
				`	"I want to explore the rest of the galaxy, but apparently parts of it are not yet implemented."`
```
The last part feels a little unnecessary and awkward.

Author: I included this because it appeared that @EjoThims wants to keep the phrase "the game"

Reviewer 2: Yes, leaving the obvious lead in to the fourth wall break is necessary to avoid it for those who wish to, as well as to highlight it for the attention it's meant to capture.

Reviewer 5: I think this can be resolved because the dialog has changed and is being decided elsewhere (in different suggestions).

> 				`	"I want to explore the rest of the galaxy, but apparently parts of it are not yet implemented in the current version of the game."`

Reviewer 2: ```suggestion
				`	"I want to explore the rest of the galaxy, but apparently not all parts of the game have been implemented yet."`
```

If you'd rather phrase it as "parts," this would be the simpler and less awkward option.

> 				`	"I want to explore the rest of the galaxy, but apparently parts of it are not yet implemented in the current version of the game."`

Reviewer 2: ```suggestion
				`	"I want to explore the entire rest of the galaxy, but apparently some parts of the game have not been implemented yet."`
```

If the primary concern is that some may leap to the conclusion that there is nothing else at all out there, you can still emphasize that not all of it is waiting without rearranging or drifting from the original sentiment, which is that the game overall still needs expansion (and help doing that), not just the galaxy itself.

This distinction is important for two reasons:

First and foremost, expansion of the galaxy is not the primary focus of improving the game overall, and currently existing stuff will still need improvement once the galaxy is filled.

Secondly, the galaxy -will- become filled, at some point (in theory, even if not in practice), meaning this changed line would then be inappropriate and need changing again once no more galactic additions are wanted.

			
			label game
			`	"Whoops," says Ijs, "now you've done it, there goes the fourth wall."`

> 			`	Katya laughs. "<first>, maybe you should volunteer to help out with creating the stories to populate it. This is an open source game after all, and it would be a sad thing for it to remain the product of only one person's imagination."`

Reviewer 4: ```suggestion
			`	Katya laughs. "If you're so eager to explore the rest of the galaxy, <first>, maybe you should volunteer to help out with creating the stories to populate it. This is an open source game after all, and it would be a sad thing for it to remain the product of only one person's imagination."`
```

> 			`	Katya laughs. "No. There are more missions and campaigns after this one, however, there also are parts of the game that have not been implemented yet. If you're so eager to explore the rest of the galaxy, <first>, maybe you should volunteer to help out with creating the stories to populate it. This is an open source game after all, and it would be a sad thing for it to remain the product of only one person's imagination."`

Reviewer 2: ```suggestion
			`	Katya laughs. "Well, there's already lots beyond human space to see, and I'm sure you can find more things to do here as well as out there. If you're so eager to explore even more of the galaxy or the adventures within it beyond that, <first>, maybe you should volunteer to help out with creating the stories to populate it. This is an open source game after all, and it would be a sad thing for it to remain the product of only one person's imagination."`
```

Reviewer 5: This sounds good, except I would turn `even more` into just `more`.

Reviewer 6: Yes, this looks good. I think either `even more` _or_ `more` work, just with different emphasis.

Reviewer 7: ```suggestion
			`	Katya laughs. "Well, there's already lots beyond human space to see, and I'm sure you can find more things to do out there as well as closer to home. If you're so eager to explore more of the galaxy, <first>, or to discover what new adventures it has to offer, maybe you should volunteer to help out with creating the stories to populate it. This is an open source game after all, and it relies on people like you to bring it life."`
```

I feel like the '...explore even more of the galaxy or the adventures within it beyond that...' section was a bit awkward so I have attempted a rewrite of that. Also a couple of other small changes, particularly changing the 'one person's imagination' bit as that is definitely no longer the case.

Reviewer 2: Yea, I'm liking that version @LazerLit  :)

Reviewer 6: Yes, that version also more adequately reflects the current dev community.

> 			`	Katya laughs. "Well, there's already lots beyond human space to see, and I'm sure you can find more things to do out there as well as closer to home. If you're so eager to explore more of the galaxy, <first>, or to discover what new adventures it has to offer, maybe you should volunteer to help out with creating the stories to populate it. This is an open source game after all, and it relies on people like you to bring it life."`

Reviewer: ```suggestion
			`	Katya laughs. "If you're so eager to explore more of the galaxy, <first>, or to discover what new adventures it has to offer, maybe you should volunteer to help out with creating the stories to populate it. This is an open source game after all, and it relies on people like you to bring it life."`
```
